docs/cost/index.md

```diff
@@ -33,43 +33,56 @@ You can read more about the Infrastructure Data Standard in [CoST Guidance Note
 
 ```
 
-## Mapping to IDS and from OCDS
-
 <!-- Note - mappings come from https://docs.google.com/spreadsheets/d/1tpXKCrNY1vUEPo6O1j-GPhxgSna7CZ5uwz_eTNLEOr8/edit#gid=2054628701 -->
 
-The following mapping tables describe:
+The following tables document two mappings:
 
-* How each element of the CoST Infrastructure Data Standard can be represented as **structured data** using the [Open Contracting for Infrastructure Data Standard](../projects/index), in the 'Mapping to OC4IDS' column.
+* The **CoST IDS to OC4IDS** mapping describes how to represent each element of the CoST IDS as structured data using OC4IDS. Use this mapping mapping if you already collect data according to the CoST IDS and you want to publish your data using OC4IDS, or if you want to make sure that your OC4IDS publication conforms to the CoST IDS.
+* The **OCDS to OC4IDS** mapping describes how to use OCDS data to populate the sections of an OC4IDS file which relate to the CoST IDS. Use this mapping if you have access to OCDS data on infrastructure contracting processes and you want to create a summary by project in OC4IDS format, or if you want to check which CoST IDS elements your OCDS data covers.
 
-* How existing OCDS data can be used to populate project-level and contracting process summary data, in the 'Mapping from OCDS' column.
+The organization of the mapping tables reflects the structure of the CoST IDS, which is described in [Getting Started](../projects/index).
 
-```eval_rst
-.. admonition:: Mapping from OCDS
-    :class: Note
+The CoST IDS also sets out a number of disclosure requirements under the heading of 'information for disclosure upon request', also known as 'reactive disclosure'. You can disclose these elements proactively using OC4IDS.
```

Comment:
I've added the clarification in https://github.com/open-contracting/infrastructure/pull/295/commits/68ca204e39c4afbbc1b154acc4d4232bb7cbfde2.

I'll leave line 12 as it is, since the reactive items are introduced with 'also'